src/Mobs/Cow.h

```diff
@@ -20,7 +20,10 @@ class cCow :
 	virtual void GetDrops(cItems & a_Drops, cEntity * a_Killer = nullptr) override;
 	virtual void OnRightClicked(cPlayer & a_Player) override;
 
-	virtual const cItem GetFollowedItem(void) const override { return cItem(E_ITEM_WHEAT); }
+	virtual void GetFollowedItems(cItems & a_Items) override
```

Comment:
I haven't seen examples of it in the source, so I just followed what appeared to be the convention.